app/User.php

```diff
@@ -28,6 +28,26 @@ class User extends Authenticatable
         'password', 'remember_token',
     ];
 
-    protected $test = 'testtest';
-    protected $test2 = 'test2test2';
+    private $test = 'testtest';
+    private $test2 = 'test2test2';
+
+    private function testWIP() {
+        $this->test = 'failed';
+        return $test;
+    }
+
+    private function unUsed() {
+        $unUse = true;
+        return null;
+        $test = 1;
+        $test2 = 2;
+    }
+
+    private function testPrefix() {
+        $unUse = false;
+        return null;
+        $test = 'yes';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "$test" which has the same name as the field declared at line 31. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://relisonar.relipasoft.com:9000/coding_rules#rule_key=php%3AS1117)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "$test" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://relisonar.relipasoft.com:9000/coding_rules#rule_key=php%3AS1481)